cli/src/commands/subgraph/commands/create.ts

```diff
@@ -47,6 +57,30 @@ export default (opts: BaseCommandOptions) => {
       }
     }
 
+    if (options.subscriptionProtocol && !isValidSubscriptionProtocol(options.subscriptionProtocol)) {
```

Comment:
3rd time duplicated code

Reply:
we are letting the user set the subprotocol in different commands like while creating, publishing and updating a subgraph or monograph so the repetition